wallet/wallet.c

```diff
@@ -2273,6 +2280,7 @@ wallet_payment_by_hash(const tal_t *ctx, struct wallet *wallet,
 					     ", msatoshi_sent"
 					     ", description"
 					     ", bolt11"
+					     ", failonionreply"
```

Comment:
does this column already exist in the database? i don't see a migration statement in this commit :eyes:

Reply:
It does indeed :+1: We just didn't display it so far.